exporter.go

```diff
@@ -91,7 +125,7 @@ var (
 	defaultScrapeURI          = getEnv("SCRAPE_URI", "http://127.0.0.1:8080/stub_status")
 	defaultSslVerify          = getEnvBool("SSL_VERIFY", true)
 	defaultTimeout            = getEnvDuration("TIMEOUT", time.Second*5)
```

Comment:
change `getEnvDuration` to `getEnvPositiveDuration`, so that it does `log.Fatalf` if the duration is not positive